app/screens/login/form.tsx

```diff
@@ -364,10 +370,17 @@ const LoginForm = ({config, extra, keyboardAwareRef, numberSSOs, serverDisplayNa
                 ref={passwordRef}
                 returnKeyType='join'
                 spellCheck={false}
-                secureTextEntry={true}
+                secureTextEntry={!isPasswordVisible}
                 testID='login_form.password.input'
                 theme={theme}
                 value={password}
+                endAdornment={<TouchableOpacity onPress={togglePasswordVisiblity}>
```

Comment:
this definitely needs to add some hitSlop as the tappable area is not that big and my fingers struggle to press it

Reply:
Agreed,hitSlop of 8 added to all sides